src/script/api/script_date_economy.hpp

```diff
@@ -5,13 +5,13 @@
  * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/** @file script_date.hpp Everything to query and manipulate date related information. */
+/** @file script_date_economy.hpp Everything to query and manipulate date related information. */
```

Comment:
Given date stuff is split over two files, it's definitely not 'everything' anymore.